### ISSUE TITLE
t-11-dovecot fails due to permission errors

### ISSUE THREAD

**Issue by foxcpp** [albertito/chasquid#28]
I get this message when running chasquid tests while building Arch Linux package:
```
t-11-dovecot ...
Fatal: mkdir(/var/lib/dovecot) failed: Permission denied
FAILED: user does not exist: no: dial unix /tmp/chasquid-dovecot-test/run/auth-userdb: connect: no such file or directory
```

Dovecot version is `2.3.19.1 (9b53102964)`.

I believe state_dir should be set in dovecot.conf to fix this. I [fixed a similar issue in maddy](https://github.com/foxcpp/maddy/commit/90f88c89cdb0330ca9840ee518d17a153b6edb0c) recently.

**Comment by albertito** [comment]
Thanks for reporting this and suggesting a fix!

It should be fixed in commit f6a39a4 which is in the `next` branch. I'll move it to `master` in a little bit.

**Comment by albertito** [comment]
The fix (735613cdf79861232d2edb25b42b69eb9a4d0a26) was merged in `master` a while ago, sorry I forgot to update this.

Thanks again!
